ada_moveit/config/real_controllers.yaml

```diff
@@ -18,6 +18,30 @@ controller_manager:
     joint_state_broadcaster:
       type: joint_state_broadcaster/JointStateBroadcaster
 
+    jaco_arm_cartesian_controller:
+      type: cartesian_twist_controller/CartesianTwistController
+
+jaco_arm_cartesian_controller:
+    ros__parameters:
+      end_effector_link: "forkTip"
+      robot_base_link: "j2n6s200_link_base"
+      ik_solver: "damped_least_squares"
+      joints:
+        - j2n6s200_joint_1
+        - j2n6s200_joint_2
+        - j2n6s200_joint_3
+        - j2n6s200_joint_4
+        - j2n6s200_joint_5
+        - j2n6s200_joint_6
+      command_interfaces:
+        - velocity
+      solver:
+        damped_least_squares:
+          alpha: 0.01
```

Comment:
Have you tried tuning this? If so, can you document what you've already done in the PR description so we know where to continue from if we need to re-tune it moving forward?

Reply:
No tuning, noted in description